Python/Product/PythonTools/PythonTools/Intellisense/ProjectAnalyzer.cs

```diff
@@ -1862,7 +1865,7 @@ class ExprWalker : PythonWalker {
                 }
 
                 yield return OutliningTaggerProvider.OutliningTagger.GetTagSpan(
-                    currentSnapshot,
+                    translator.TextBuffer.CurrentSnapshot,
```

Comment:
Yeah, I'll hoist that to a local at the start of the function, the important thing is that we get the current snapshot when we entered this function